tests/API/Integration/DepartmentsTests.cs

```diff
@@ -99,6 +99,49 @@ public async Task GetAuditorParksMemberUnits()
 				Assert.That(actual.First().Id, Is.EqualTo(expected.First().Id));
 				Assert.That(actual.First().Parent.Id, Is.EqualTo(expected.First().Parent.Id));
 			}
+
+			[Test]
+			public async Task GetsDistinctMemberUnits()
+			{
+				// Add Chris as a second member of the Auditor team.
+				var db = Database.PeopleContext.Create(Database.PeopleContext.LocalDatabaseConnectionString);
+				var chris = new Person
+				{
+					Netid = "ctraeger",
+					Name = "Traeger, Chris",
+					NameFirst = "Chris",
+					NameLast = "Traeger",
+					Position = "Sr. Auditor",
+					Location = "Pawnee",
+					Campus = "Pawnee",
+					CampusPhone = "5-6789",
+					CampusEmail = "ctraeger@auditz.com",
+					Notes = "",
+					PhotoUrl = "",
+					Responsibilities = Responsibilities.None,
+					DepartmentId = TestEntities.Departments.AuditorId,
+					UnitMemberships = new List<UnitMember>()
+				};
+				
+				var chrisAuditor = new UnitMember
+				{
+					UnitId = TestEntities.Units.AuditorId,
```

Comment:
Same question here `UnitId=TestEntities.Units.Auditor.Id`?  (Sorry, If I'm missing something)

Reply:
Ditto. `TestEntities.Units.Auditor.Id` is set using `TestEntities.Units.AuditorId`.

Once again, feel free to change it for legibility if you have a preference.

Reply:
Thanks, again.